app/controllers/staffs/dashboard_controller.rb

```diff
@@ -10,11 +9,78 @@ def index
       @download = Quiz::LESSON_VALUES.map { |n| ["Lesson #{n}", n] }
     end
 
+    def import
+      @results = []
+    end
+
+    def import_reader
+      @logins = params[:logins]
+      @results = @logins.split
+                        .reject { |login| Reader.find_by_login login }
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.